owncloud/indico_owncloud/client/index.js

```diff
@@ -9,6 +9,8 @@
 window.setupOwncloudFilePickerWidget = ({filepickerUrl, fieldId}) => {
   window.addEventListener('message', message => {
     const iframe = document.querySelector('#owncloud_filepicker-file-picker');
+    const submitButton = document.querySelectorAll('input[type=submit]')[0];
```

Comment:
```suggestion
    const submitButton = document.querySelector('input[type=submit]');
```

However, I believe this is too generic since it can match *any* submit button on that page...